docroot/modules/custom/foia_file/foia_file.drush.inc

```diff
@@ -75,7 +75,7 @@ function drush_foia_file_file_entity_updater($scanResult) {
 
     // Update file entity based upon status.
     $fileEntity = FileEntity::load($fid);
-    if ($fileEntity && $fileEntity->get('field_virus_scan_status') === 'scan') {
+    if ($fileEntity && $fileEntity->get('field_virus_scan_status')->getString() === 'scan') {
```

Comment:
I think `->value` should work as well, which is what we use elsewhere. Might want to try that so that we're consistent in our approach to accessing field values. If that doesn't work let's use the `getString` method.